### ISSUE TITLE
Please Choose a License

### ISSUE THREAD

**Issue by chickenandpork** [markusdosch/gopherlol#1]
Are you willing to nominate a license such as MIT license for your project?  I cannot see one in code nor as a meta-data to the project (although Github has changed its interface, I could be committing a piloting error here.  I thought there was a permissive license, but I don't see one now).

May I add a LICENSE file so that re-use in various organizations is relatively free of complexity?

Thanks!

**Comment by markusdosch** [comment]
Thanks for pointing it out, forgot about it when creating the project...
I now added a `LICSENSE` file (MIT license). What do you think - is licensing alright now? :) 

https://github.com/markusdosch/gopherlol/blob/master/LICENSE

**Comment by markusdosch** [comment]
(by the way, awesome that you forked & extended the project! 🎉)

**Comment by chickenandpork** [comment]
That is optimal, as best as it can be.

I made some changes as well to make it easer to simply add files to extend -- I have an example of extending it as well.  The intent was to enable just adding files so that merged have lower risk of merge-error (git's not that smart, so separate files).

Let me pull in your license, and I'll open up the example extension.  I think that's the one I have experimented with in bazel to build a running image that gets pushed to k8s.
